compiler/semstmts.nim

```diff
@@ -431,7 +431,53 @@ proc setVarType(c: PContext; v: PSym, typ: PType) =
         "; new type is: " & typeToString(typ, preferDesc))
   v.typ = typ
 
+proc semLowerLetVarCustomPragma(c: PContext, a: PNode, n: PNode): PNode =
+  var b = a[0]
+  if b.kind == nkPragmaExpr:
+    if b[1].len != 1:
+      # we could in future support pragmas w args eg: `var foo {.bar:"goo".} = expr`
+      return nil
+    let nodePragma = b[1][0]
+    # see: `singlePragma`
+    if nodePragma.kind notin {nkIdent, nkAccQuoted}:
```

Comment:
I think there can also be ``nkOpenSymChoice``, ``nkClosedSymChoice``, see ``considerQuotedIdent``.

Reply:
please provide an example that would require this; in any case, this could be added in future PR's as future enhancement (more things (not less) becoming possible with var/let user pragmas) without breaking code.